src/Controller/API/AamcMethods.php

```diff
@@ -63,6 +193,45 @@ public function post(
         '/{id}',
         methods: ['PUT']
     )]
+    #[OA\Put(
+        path: '/api/{version}/aamcmethods/{id}',
+        summary: 'Update an AAMC method.',
+        parameters: [
+            new OA\Parameter(name: 'version', description: 'API Version', in: 'path'),
+            new OA\Parameter(name: 'id', description: 'id', in: 'path'),
+            new OA\Parameter(
+                name: 'body',
+                in: 'body',
+                required: true,
+                schema: new OA\Schema(
+                    properties: [
+                        new OA\Property(
+                            'aamcMethod',
+                            ref: new Model(type: AamcMethodDTO::class),
+                            type: 'object'
+                        )
+                    ],
+                    type: 'object',
+                )
+            )
+        ]
+    )]
+    #[OA\Response(
```

Comment:
`PUT` can also return 201 if a new object is created just like `POST`

Reply:
added.

Reply:
:heavy_check_mark: